spec/controllers/comments_controller_spec.rb

```diff
@@ -15,19 +15,23 @@
       end
 
       it "sends a comment email to approvers and observers" do
+        ENV["DISABLE_EMAIL"] = nil
```

Comment:
How about using RSpec metadata for this, just like we do in feature specs with the `js: true` flag (which tells Capybara to use Poltergeist)?